test/blocks/merch-card/mocks/acrobat-card.html

```diff
@@ -0,0 +1,63 @@
+                    <span class="price-unit-type">per license</span>
+                    <span class="price-tax-inclusivity disabled"></span>
+                  </span>
+                </span></li>
+                <li><strong><a is="checkout-link" data-checkout-workflow="UCv3" data-checkout-workflow-step="email" data-quantity="1" data-wcs-osi="6WK1gybjBe2EKcq0HI0WvbsoiKOri2yRAwS9t_kGHoE" class="con-button blue placeholder-resolved" href="https://commerce.adobe.com/store/email?items%5B0%5D%5Bid%5D=DF7F553163565F8621D8AE91D978F354&amp;cli=adobe_com&amp;ctx=fp&amp;co=US&amp;lang=en">Buy now</a></strong></li>
+              </ul>
+            </li>
+            <li>Annual, one-time payment
+              <ul>
+                <li>Best Offer</li>
+                <li><span is="inline-price" data-display-per-unit="true" data-display-recurrence="true" data-display-tax="false" data-force-tax-exclusive="true" data-wcs-osi="gr3e95wowwDvLJyphdXmBf9-vTub0fhbdxQfGJ7tdhA" class="placeholder-resolved">
+                    <span class="price" aria-label="US$149.88 per year per license">
+                      <span class="price-currency-symbol">US$</span>
+                      <span class="price-currency-space disabled"></span>
+                      <span class="price-integer">149</span>
+                      <span class="price-decimals-delimiter">.</span>
+                      <span class="price-decimals">88</span>
+                      <span class="price-recurrence">/yr</span>
+                      <span class="price-unit-type">per license</span>
+                      <span class="price-tax-inclusivity disabled"></span>
+                    </span>
+                </span></li>
+                <li><strong><a is="checkout-link" data-checkout-workflow="UCv3" data-checkout-workflow-step="email" data-quantity="1" data-wcs-osi="gr3e95wowwDvLJyphdXmBf9-vTub0fhbdxQfGJ7tdhA" class="con-button blue placeholder-resolved" href="https://commerce.adobe.com/store/email?items%5B0%5D%5Bid%5D=5D023AC48C69229033C8ECE450A920F0&amp;cli=adobe_com&amp;ctx=fp&amp;co=US&amp;lang=en">Buy now</a></strong></li>
+                <li>New Description</li>
+              </ul>
+            </li>
+            <li>Monthly, without commitment
+              <ul>
+                <li><span is="inline-price" data-display-per-unit="true" data-display-recurrence="true" data-display-tax="false" data-force-tax-exclusive="true" data-wcs-osi="BWGlzrQG6jgkf-_zPJm55M5QpAyb4skYi05BizQIJ3U" class="placeholder-resolved" ><span class="price" aria-label="US$29.99 per month per license"><span class="price-currency-symbol">US$</span><span class="price-currency-space disabled"></span><span class="price-integer">29</span><span class="price-decimals-delimiter">.</span><span class="price-decimals">99</span><span class="price-recurrence">/mo</span><span class="price-unit-type">per license</span><span class="price-tax-inclusivity disabled"></span></span></span></li>
```

Comment:
I think you should use unprocessed placeholders, here it is already resolved (class`="placeholder-resolved"`)
same for links, they should not have href set yet.

Reply:
doesn't it mean we'd need to copy all the mocking from merch and load merch itself?
otherwise they don't resolve and look quite ugly.
but copying all that stuff seemed also a bit too much to me, especially taking into account that merch-card itself also doesn't use any of the real placeholders